pkg/agent/flowexporter/exporter/exporter.go

```diff
@@ -193,7 +198,29 @@ func (exp *flowExporter) initFlowExporter(collectorAddr string, collectorProto s
 	// TODO: This code can be further simplified by changing the go-ipfix API to accept
 	// collectorAddr and collectorProto instead of net.Addr input.
 	var expInput exporter.ExporterInput
-	if collectorProto == "tcp" {
+	if exp.enableTLSToFlowAggregator {
+		// if CA certificate, client certificate and key do not exist during initialization,
```

Comment:
Is this comment applicable? I do not see any checks if CA certificate, client certificate and key exists or not.

Reply:
Hi @srikartati - I asked for this comment: if one of `getCACert` or `getClientCertKey` fails because of a K8s resource which does not exist yet, this initialization will fail and we will retry in the next export cycle. The certificates are created by the flow aggregator, and it may not be running yet when this code executes for the first time(s).

Reply:
Agreed. Yes, the exporter has to try again. Even if there is an old K8s resource (Flow Aggregator restart scenario), I think we still need to get the updated resource before connecting to Flow Aggregator.

I misconstrued exist as exist in the flowExporter struct, so got slightly confused.